main.py

```diff
@@ -310,12 +314,15 @@ def load_ref_data_rds(self):
             # Loading of asn with push_datapackage takes more then 2 hours
             # So have to download localy and sasve (takes ~5 seconds)
             if 'asn' not in url:
-                push_datapackage(descriptor=url,backend='sql',engine=conn)
+                push_datapackage(descriptor=url, backend='sql', engine=conn)
             else:
                 dp = datapackage.DataPackage(url)
                 # local path will be returned if not found remote one (fot tests)
                 url = dp.resources[0].remote_data_path or dp.resources[0].local_data_path
-                urllib.urlretrieve(url, join(self.tmpdir, 'asn.csv'))
+                r = requests.get(url)
```

Comment:
can we keep `urllib.urlretrieve` ? as for tests we load datapackage.json from `/test/fixtures/...`

This is the reason travis is failing